clvm/more_ops.py

```diff
@@ -49,16 +50,16 @@
 )
 
 
-def malloc_cost(cost, atom):
-    return cost + len(atom.atom) * MALLOC_COST_PER_BYTE, atom
+def malloc_cost(cost, atom: SExp):
+    return cost + len(atom.as_atom()) * MALLOC_COST_PER_BYTE, atom
```

Comment:
Ditto this, can be reverted. (I used to use `.as_atom` and `.as_pair` exclusively, but decided to switch to using `.atom` and `.pair` when possible so that we can use `CLVMObject` in more places.)